pkg/model/constants.go

```diff
@@ -12,7 +12,7 @@ const (
 	PostgresqlServiceName                = ApplicationName + "-postgresql"
 	PostgresqlImage                      = "postgres:9.5"
 	KeycloakImage                        = "quay.io/keycloak/keycloak:7.0.1"
-	KeycloakInitContainerImage           = "quay.io/keycloak/extensions-init-container:master"
+	KeycloakInitContainerImage           = "quay.io/keycloak/keycloak-init-container:latest"
```

Comment:
```suggestion
	KeycloakInitContainerImage           = "quay.io/keycloak/keycloak-init-container:master"
```
This is better as master based on the conversation in chat. @pb82 pointed out that it is a bit safer than latest because it only gets updated once something is merged into master.

Reply:
Fixed. Thanks!